python/torch_mlir_e2e_test/test_suite/basic.py

```diff
@@ -1360,6 +1360,54 @@ def AddCDivModule_basic(module, tu: TestUtils):
 # ==============================================================================
 
 
+class AddCMul_Module(torch.nn.Module):
```

Comment:
No need to add these tests. These underscore ops will get replaced with the corresponding non-underscore variant before reaching any of the backends, so the tests for the non-underscore variants are sufficient.